src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -88,28 +95,31 @@ stop_lon | Quay/Centroid/Location | see (1) below
 parent_station | | stop_id of the corresponding `stop_area`, see (2) below.
 stop_timezone | | fixed value `Europe/Paris`
 equipment_id | | This value will be self generated by the connector if an equipment object is necessary. See (2) below.
+fare_zone_id | Quay/tariffZones/TariffZoneRef/@ref | If the `tariffZones` does not exists or is empty, this field will be empty. If more than one fare zone are available, the first one will be considered. The fare zone needs to be extract from the content of the `@ref` attribute, using the 3rd field using a `:` separator. If this 3rd field is not an integer, the resulting `fare_zone_id` will be empty.
+
+**(1) Definition of stop_lat et stop_lon:**
 
-(1) Definition of stop_lat et stop_lon:
-The `Quay/Centroid/Location` node contains a position (X, Y) with an EPSG described, for example: 
-`<gml:pos srsName="EPSG:2154">662233.0 6861519.0</gml:pos>`
-This coordinate is to be converted to a WGS84 coordinate (EPSG:4326).
+The node `Quay/Centroid/Location` declares the position (X, Y) of the stop_point with an explicit EPSG always set to 2154. This coordinate will be converted to the NTFS WGS84 projection (EPSG:4326).
+Exemple of Netex declaration:
+><gml:pos srsName="EPSG:2154">662233.0 6861519.0</gml:pos>
 
-(2) Definition of the parent_station:
-A ZDL `StopPlace` includes the references of it's ZDE in `StopPlace/quays/QuayRef` nodes. The parent_station of a `stop_point` can be either: 
-* a ZDL `stop_area` referencing its `Quay`
-* an LDA `stop_area` contains it's ZDL references as complementary object_codes, each ZDL referencing its `Quay`
+**(2) Definition of the parent_station:**
+Linking a stop_area to a stop_point requires to navigate from lower `Quay` to upper `StopPlace` nodes:
+- the operator `Quay` node (aka ZDEp) references the PTA `Quay` node (aka ZDEr) with the attribute `Quay/@derivedFromObjectRef`,
+- the PTA `Quay` (aka ZDEr) references the monomodal `StopPlace` node (aka ZDL) with the attribute `Quay/ParentZoneRef/@ref`
+- the monomodal `StopPlace` node (aka ZDL) references the multimodal `StopPlace` (aka LDA, the actual `stop_area`) with the attribute `StopPlace/ParentSiteRef/@ref`.
```

Comment:
Should we have something like
```
... unless there is no multimodal parent, then we use the monomodal as the stop_area
```

Reply:
fixed, and I've added the creation of a missing stop_area (should be in generic behavior, but added here for now)